source/lbaas.rst

```diff
@@ -0,0 +1,521 @@
+  | name                | 90_member                            |
+  | operating_status    | NO_MONITOR                           |
+  | project_id          | a3a9af91b9e547739bfcb02cc2acded0     |
+  | protocol_port       | 90                                   |
+  | provisioning_status | PENDING_CREATE                       |
+  | subnet_id           | None                                 |
+  | updated_at          | None                                 |
+  | weight              | 1                                    |
+  | monitor_port        | None                                 |
+  | monitor_address     | None                                 |
+  +---------------------+--------------------------------------+
+
+As a simple mockup we have the commands shown below running on each of the member servers, they
+will send a response when a connection is received on the listening port. Make sure that you
+replace the PORT variable with the correct value for each member server.
+
+.. code-block:: bash
+
+  export MYIP=$(/sbin/ifconfig eth0 |grep 'inet addr'|awk -F: '{print $2}'| awk '{print $1}');
+  export PORT="80"
+  sudo nc -lk -p ${PORT} -e echo -e "HTTP/1.1 200 OK\r\n$(date)\r\n\r\n\tHi this is port ${PORT} on ${MYIP}\n\n"
+
+To test, telnet to both of the ports at VIP of the listener, in this case 10.0.0.3, in response you
+should expect to get an appropriate response for the targeted port indicating that the correct
+server has responded to the request.
+
+.. code-block:: console
+
+  $ telnet 10.0.0.3 80
+  Trying 10.0.0.3...
+  Connected to 10.0.0.3.
+  Escape character is '^]'.
+  HTTP/1.1 200 OK
+  Thu Nov  9 01:25:08 UTC 2017
+
+    Hi this is port 80 on 10.0.0.4
+
+  Connection closed by foreign host.
+
+
+  $ telnet 10.0.0.3 90
+  Trying 10.0.0.3...
+  Connected to 10.0.0.3.
+  Escape character is '^]'.
+  HTTP/1.1 200 OK
+  Thu Nov  9 01:25:55 UTC 2017
+
+    Hi this is port 90 on 10.0.0.12
+
+
+  Connection closed by foreign host.
+
+
+What is L7 load balancing?
+==========================
+
+Layer 7 load balancing takes its name from the OSI model, indicating that the load balancer
+distributes requests to back-end pools based on layer 7 (application) data. Layer 7 load balancing
+s also known as “request switching,” “application load balancing,” “content based routing or
+switching,”
+
+A layer 7 load balancer consists of a listener that accepts requests on behalf of a number of
+back-end pools and distributes those requests based on policies that use application data to
+determine which pools should service any given request. This allows for the application
+infrastructure to be specifically tuned/optimized to serve specific types of content. For example,
+one group of back-end servers (pool) can be tuned to serve only images, another for execution of
+server-side scripting languages like PHP and ASP, and another for static content such as HTML, CSS,
+and JavaScript.
```

Comment:
I'd prefer a more realistic example here, such as "mydomain.nz/login or a subdomain login.mydomain.nz will be routed to a back-end pool running an identity provider and authentication system, while mydomain.nz/shop or shop.mydomain.nz will be routed to a commerce application".